internal/assertion/setup.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package assertion
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"k8s.io/klog"
+
+	"sigs.k8s.io/krew/internal/environment"
+	"sigs.k8s.io/krew/internal/installation"
+)
+
+const (
+	instructionWindows = `To complete the installation, you need to add the "%USERPROFILE%\.krew\bin" directory to your PATH environment variable`
+	instructionZsh     = `~/.zshrc:
+
+    export PATH="${PATH}:${HOME}/.krew/bin"`
+	instructionBash = `~/.bash_profile or ~/.bashrc:
+
+    export PATH="${PATH}:${HOME}/.krew/bin"`
+	instructionFish = `config.fish:
+
+    set -gx PATH $PATH $HOME/.krew/bin`
+	instructionGeneric = `~/.bash_profile, ~/.bashrc, or ~/.zshrc:
+
+    export PATH="${PATH}:${HOME}/.krew/bin"`
+)
+
+func CheckSetup(w io.Writer, paths environment.Paths) {
+	if isSetupGood(paths) {
+		klog.V(4).Info("Krew setup looks good")
+		return
+	}
+
+	red := color.New(color.FgRed).SprintfFunc()
+	if installation.IsWindows() {
+		fmt.Fprintf(w, "%s: %s\n\n", red("ATTENTION"), instructionWindows)
```

Comment:
looks like we're duplicating this line below. maybe come up with a `msg string` and print that at the end of the method?